### ISSUE TITLE
Repo has untracked source files: ['klippy/extras/led_effect.py']

### ISSUE THREAD

**Issue by JNSmith1717** [julianschill/klipper-led_effect#161]
like it says under Update Manager under klipper it has "Repo has untracked source files: ['klippy/extras/led_effect.py']"

**Comment by JNSmith1717** [comment]
<img width="372" alt="Screenshot 2023-10-20 124142" src="https://github.com/julianschill/klipper-led_effect/assets/69747148/c8c7cb36-ac95-4e25-8a8f-74d1018404e3">

**Comment by ayesim** [comment]
it is normal. klipper has additional files in klipper installation folder and warning you. it's not an error message.

**Comment by JNSmith1717** [comment]
Yeah i know just annoying, is there a way to get rid of this message?  Also is there a way to uninstall the led_effect?  If this continues to show this I would rather just uninstall the led_effect.

**Comment by julianschill** [comment]
There is no way to get rid of that message. It's just how the plugin extends Klipper. Same method is used for all Klipper "plugins". To get rid of the message you have to uninstall it. Just delete the file `~/klipper/klippy/extras/led_effect.py`.

**Comment by JNSmith1717** [comment]
> There is no way to get rid of that message. It's just how the plugin extends Klipper. Same method is used for all Klipper "plugins". To get rid of the message you have to uninstall it. Just delete the file `~/klipper/klippy/extras/led_effect.py`.

Thank you, I think I will just delete, nothing agaisnt what you did, I think its a great Idea but honestly I don't even use it.  I wish I could have found none RGB lights for the stealth burner.

**Comment by JNSmith1717** [comment]
Well i deleted ~/klipper/klippy/extras/led_effect.py and it still says the Repo has untrack source files for this, is there a fix for this?

**Comment by julianschill** [comment]
Try refreshing the repos (there is a button in Mainsail) or rebooting and reloading the page. Also make sure it is led_effect.py it reports as untracked and not some other plugin.
